tests/Unit/ParallelAlgorithms/Events/Test_ObserveFields.cpp

```diff
@@ -304,6 +310,19 @@ void test_observe(const std::unique_ptr<ObserveEvent> observe) noexcept {
   const std::string element_name = get_output(element_id);
   const Mesh<volume_dim> mesh(5, Spectral::Basis::Legendre,
                               Spectral::Quadrature::GaussLobatto);
+
+  std::array<Matrix, volume_dim> interpolation_matrices{};
+  if (interpolating_mesh) {
+    const auto slices = mesh.slices();
+    const auto interpolating_slices = interpolating_mesh->slices();
+    for (size_t i = 0; i < volume_dim; i++) {
+      const auto& target_points =
+          Spectral::collocation_points(interpolating_slices.at(i));
+      interpolation_matrices.at(i) =
+          Spectral::interpolation_matrix(slices.at(i), target_points);
```

Comment:
This section of the test appears to be copy-pasted from the code it is testing.  Can you do an independent test?  Maybe construct linear data and check that is is interpolated correctly.